packages/client/hmi-client/src/types/common.ts

```diff
@@ -87,3 +87,15 @@ export type Tab = {
 	assetId?: string;
 	assetType?: ProjectAssetTypes | 'overview' | 'workflow' | '';
 };
+
+export enum AcceptedTypes {
+	PDF = 'application/pdf',
+	JPG = 'image/jpg',
+	JPEG = 'image/jpeg',
+	PNG = 'image/png'
+}
+
+export interface PDFExtractionResponseType {
```

Comment:
Re: your recent change to take PDFExtractionResponseType out of Types.ts...

If you create a java class for PDFExtractionResponseType and then annotate it with @TSModel you will get a POJO created for the back end to use and then the ts implementation would come for free :)

Reply:
Thanks @dvince2 .  Yeah that was my original thought but I'm not quite sure this actually belongs so I just kept it simple!